README.md

```diff
@@ -1,4 +1,8 @@
-# Printing using LPD with gRPC and Protocol Buffers
+# Core-v4 RPC
+
+This repository contains all of the RPC's used in SCE's Core-v4 website. 
+
+## Printing using LPD with gRPC and Protocol Buffers
```

Comment:
what do you think abut making the sub headers all `##`? see [the Visual Studio Code README](https://github.com/microsoft/vscode/blob/master/README.md)

Reply:
I think its better if we have a hierarchical design for headers